tools/gn

```diff
@@ -646,7 +646,9 @@ def to_gn_args(args):
   if args.unoptimized and args.target_os == 'android' and args.android_cpu == 'arm64':
     enable_vulkan_validation = True
 
-  if enable_vulkan_validation:
+  if enable_vulkan_validation or (args.enable_impeller_vulkan and
```

Comment:
With the changes to the GN build for the validation layers .so, does the API level for the whole build still need to be bumped up to 26 down on line 654?